src/pages/Account/Login.tsx

```diff
@@ -38,9 +39,18 @@ export const Login: React.FC<LoginProps> = ({
 
   const [formToDisplay, setFormToDisplay] = useState(formType);
 
-  const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn(
-    venue.samlAuthProviderId
-  );
+  const {
+    isShown: isLoginLoadingShown,
+    show: showLoginLoading,
+    hide: hideLoginLoading,
+  } = useShowHide(false);
+
+  const { signInWithSAML, hasSamlAuthProviderId } = useSAMLSignIn({
+    samlAuthProviderId: venue.samlAuthProviderId,
+    samlProfileMappings: venue.samlProfileMappings,
+    showLoginLoading,
+    hideLoginLoading,
+  });
```

Comment:
What's the point of having the `useShowHide` out here, just to pass it's values into `useSAMLSignIn`?

Surely we should just track the loading state within that provider itself, and return the appropriate status?

I would probably also destructure the keys off `venue` too:

Something like:

```suggestion
  const { samlAuthProviderId, samlProfileMappings } = venue;
  
  const { signInWithSAML, hasSamlAuthProviderId, isSAMLLoginLoading } = useSAMLSignIn({
    samlAuthProviderId,
    samlProfileMappings,
  });
```

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1667/commits/c77cd255fb597da0dd2611bc66089e77dbe0e59f

Reply:
Since the logic change, the changes are drastic